src/v/cluster/partition_probe.cc

```diff
@@ -102,6 +84,16 @@ void replicated_partition_probe::setup_metrics(const model::ntp& ntp) {
           [this] { return _records_fetched; },
           sm::description("Total number of records fetched"),
           labels),
+        sm::make_total_bytes(
+          "bytes_produced",
```

Comment:
```suggestion
          "bytes_produced_total",
```

There's a Prometheus naming convention to suffix all counters with `_total`. I'm unsure if it is too late, but I thought I'd leave it here and you can decide then

https://prometheus.io/docs/practices/naming/

> ...should have a suffix describing the unit, in plural form. Note that an accumulating count has total as a suffix, in addition to the unit if applicable.
http_request_duration_seconds
node_memory_usage_bytes
http_requests_total (for a unit-less accumulating count)
process_cpu_seconds_total (for an accumulating count with unit)
foobar_build_info (for a pseudo-metric that provides metadata about the running binary)